connector/authproxy/authproxy.go

```diff
@@ -14,16 +14,24 @@ import (
 
 // Config holds the configuration parameters for a connector which returns an
 // identity with the HTTP header X-Remote-User as verified email.
-type Config struct{}
+type Config struct {
+	HeaderName string   `json:"headerName"`
+	Groups     []string `json:"groups"`
+}
 
 // Open returns an authentication strategy which requires no user interaction.
 func (c *Config) Open(id string, logger log.Logger) (connector.Connector, error) {
-	return &callback{logger: logger, pathSuffix: "/" + id}, nil
+	if c.HeaderName == "" {
```

Comment:
Note that this actually modifies the config. I think it would be better to create a local variable for the header and fall back to the default if it's empty.

Reply:
ok, I've used a default value in the callback directly without modifying the config.